graph/src/blockchain/types.rs

```diff
@@ -81,10 +81,27 @@ pub struct BlockPtr {
 
 impl CheapClone for BlockPtr {}
 
+impl stable_hash_legacy::StableHash for BlockPtr {
+    fn stable_hash<H: stable_hash_legacy::StableHasher>(
+        &self,
+        mut sequence_number: H::Seq,
+        state: &mut H,
+    ) {
+        stable_hash_legacy::utils::AsBytes(self.hash.0.as_ref())
+            .stable_hash(sequence_number.next_child(), state);
```

Comment:
I see that you added a destructure to the one struct that I put the comment on, but this comment applies to all structs/tuples implementing `StableHash` (modern and legacy)

Reply:
yep sure I'll add those, totally forgot

Reply:
fixed